src/pages/cli/graphql-transformer/searchable.mdx

```diff
@@ -133,13 +133,13 @@ Here is a complete list of searchable operations per GraphQL type supported as o
 
 ### Backfill your Elasticsearch index from your DynamoDB table
 
-The following Python [script](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py) creates an event stream of your DynamoDB records and sends them to your Elasticsearch Index. This will help you backfill your data should you choose to add `@searchable` to your @model types at a later time.
+The following Python [script](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py) creates an event stream of your DynamoDB records and sends them to your OpenSearch Index. This will help you backfill your data should you choose to add `@searchable` to your @model types at a later time.
```

Comment:
Are the `graphql-elasticsearch-transformer` package name references changing (`opensearch`)?

Reply:
No.